### PR TITLE
Polish language pack adjusted to recent changes in `main`

### PR DESCRIPTION
**Pull request by strugman**
*Issue #, if available:* None

*Description of changes:*
Polish translations file

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by wlee221 — COMMENTED** [review]
Hello @strugman, thanks for your contribution! Can you do us a favor and add an entry about Polish support in our [docs](https://ui.docs.amplify.aws/components/authenticator#internationalization-i18n)?

The file is located in https://github.com/aws-amplify/amplify-ui/blob/main/docs/src/pages/components/authenticator/index.page.mdx. You would add `pl - Polish` to Line 376-382, which lists supported languages in alphabetical order.

**Comment by strugman** [comment]
> Hello @strugman, thanks for your contribution! Can you do us a favor and add an entry about Polish support in our [docs](https://ui.docs.amplify.aws/components/authenticator#internationalization-i18n)?
> 
> The file is located in https://github.com/aws-amplify/amplify-ui/blob/main/docs/src/pages/components/authenticator/index.page.mdx. You would add `pl - Polish` to Line 376-382, which lists supported languages in alphabetical order.

Hiya
no problem - I've just pushed a change. Please note, languages were not listed there in alphabetical order, so I've reordered them.

File: docs/src/pages/components/authenticator/index.page.mdx [hunk 1]
```diff
@@ -373,13 +373,14 @@ The following example customizes these slots with:
...
 - `en` – English (default)
-- `zh` – Chinese
+- `es` – Spanish
 - `fr` – French
-- `de` – German
 - `it` – Italian
 - `ja` – Japenese
-- `es` – Spanish
+- `pl` - Polish
+- `zh` – Chinese
```
Comment (wlee221) [inline docs/src/pages/components/authenticator/index.page.mdx:383]: Ah, my bad, I meant alphabetical with the language names to the right (with English at the top). I'll readjust it, sorry for poorly explaining it!

**Review by wlee221 — APPROVED** [review]
LGTM, thanks @strugman!

**Pull request merged**